lib/dotcom_web/components/live_components/itinerary_detail.ex

```diff
@@ -0,0 +1,105 @@
+defmodule DotcomWeb.Components.LiveComponents.ItineraryDetail do
+  @moduledoc """
+  The section of the trip planner page that shows the map and
+  the summary or details panel
+  """
+
+  use DotcomWeb, :live_component
+
+  import DotcomWeb.Components.TripPlanner.Leg
+
+  alias Dotcom.TripPlan.PersonalDetail
+
+  @impl true
+  def mount(socket) do
+    {:ok, socket |> assign(:selected_trip_index, 0)}
+  end
+
+  @impl true
+  def render(%{itineraries: itineraries, selected_trip_index: selected_trip_index} = assigns) do
+    assigns =
+      assigns
+      |> assign(:selected_itinerary, Enum.at(itineraries, selected_trip_index))
+      |> assign(:border_classes, "border border-brand-primary rounded")
```

Comment:
Take these and the other classes assigns out, as far as I can tell you add these elsewhere

Reply:
Oops!